src/Compilers/CSharp/Test/Emit/Emit/EmitErrorTests.cs

```diff
@@ -106,7 +106,11 @@ public static void Main()
     }
 }
 ";
-            VerifyEmitDiagnostics(source2, compilation1);
+            var compilation2 = CompileAndVerify(
+                source2,
+                new[] { new CSharpCompilationReference(compilation1) },
+                verify: false);
+            compilation2.Emit(expectedOutput: null, manifestResources: null, peVerify: false, expectedSignatures: null);
```

Comment:
There is no result to check here, the return is `void`.  This `Emit` API signals failure by throwing an exception.  In both cases we are just verifying that we can emit without throwing.  Without the fix both cases will throw an exception and hence fail the test.